dev/run-tests.py

```diff
@@ -546,7 +546,8 @@ def main():
         hadoop_version = os.environ.get("AMPLAB_JENKINS_BUILD_PROFILE", "hadoop2.7")
         test_env = "amplab_jenkins"
         # add path for Python3 in Jenkins if we're calling from a Jenkins machine
-        os.environ["PATH"] = "/home/anaconda/envs/py3k/bin:" + os.environ.get("PATH")
+        # TODO(sknapp):  s/py36/py3k before merging!
```

Comment:
OK by me; does this need to change before you merge?

Reply:
yeah...  this is so i can test against the py36 conda environment on the jenkins workers.  right before i merge, i'll pause jenkins, update the conda env name, push my revert and then let jenkins build again.

folks will need to update their PRs post-merge as well.

i'm waiting on @BryanCutler and the arrow 0.12.1 release before doing any of this, however.  probably early to mid next week.

there will be a detailed email going out to dev@ about this on monday.

Reply:
this also reminds me that one day, i would love to go through everything and remove all mention of amplab.  that project has been over since 2016!  :)

Reply:
quick update:  `py36` will actually remain the path for this python env on `master`...